src/libraries/Microsoft.Extensions.Primitives/src/StringValues.cs

```diff
@@ -242,9 +242,7 @@ static string GetJoinedStringValueFromArray(string[] values)
                     }
                 });
 #else
-#pragma warning disable CS0618
-                var sb = new InplaceStringBuilder(length);
-#pragma warning restore CS0618
+                var sb = new StringBuilder(length);
```

Comment:
Should we use ValueStringBuilder here instead?

Reply:
#39163 looks like a dupe of this PR.

This entire `#if #else` was removed there as it's checking `NETCOREAPP || NETSTANDARD2_1`